src/components/EmailTemplateClient.tsx

```
@@ -0,0 +1,55 @@
+'use client'
+
+// organize-imports-ignore
+import React from 'react'
+import type { RenderEmailTemplateProps } from '../utils/renderEmailTempalte.js'
```

ESLint

[error] Missed spacing between "react" and "../utils/renderEmailTempalte.js" imports.

(perfectionist/sort-imports)